### PR TITLE
[STORY] Quota editing in GLU

### PR DESCRIPTION
**Pull request by MarcoKramer**
AB#46597

Related PRs:
https://github.com/NIPOSoftwareBV/Nfield-Manager-SurveysAndSampleManagement/pull/425
https://github.com/NIPOSoftwareBV/project-glu-api/pull/1931
https://github.com/NIPOSoftwareBV/project-glu-client/pull/2288
https://github.com/NIPOSoftwareBV/nfield-source/pull/3381
https://github.com/NIPOSoftware/Nfield-SDK/pull/151

### CODE REVIEW

File: Nfield.Quota/Builders/QuotaFrameBuilder.cs [hunk 1]
```diff
@@ -49,23 +49,26 @@ public QuotaFrameBuilder Target(int? target)
         public QuotaFrameBuilder VariableDefinition(
             string variableName,
             string odinVariableName,
-            IEnumerable<string> levelNames)
+            IEnumerable<string> levelNames,
+            bool? isSelectionOptional = null)
```
Comment (joukevandermaas) [inline Nfield.Quota/Builders/QuotaFrameBuilder.cs]: Rather than a nullable bool, could we use an enum instead?

Comment (serifozcan) [reply Nfield.Quota/Builders/QuotaFrameBuilder.cs]: We'll do it

**Comment by serifozcan** [comment]
@marcoNipo Do you mean we need to update major version?

**Comment by marcoNipo** [comment]
@serifozcan this look like a minor [version](https://semver.org/) update material. Too late anyway

**Pull request merged**